diff --git a/library/daloradius.conf.php b/library/daloradius.conf.php
index 4f1e893..aa86c93 100644
--- a/library/daloradius.conf.php
+++ b/library/daloradius.conf.php
@@ -26,11 +26,11 @@
 $configValues['DALORADIUS_VERSION'] = '0.9-9';
 $configValues['FREERADIUS_VERSION'] = '2';
 $configValues['CONFIG_DB_ENGINE'] = 'mysqli';
-$configValues['CONFIG_DB_HOST'] = 'localhost';
-$configValues['CONFIG_DB_PORT'] = '3306';
-$configValues['CONFIG_DB_USER'] = 'root';
-$configValues['CONFIG_DB_PASS'] = '';
-$configValues['CONFIG_DB_NAME'] = 'radius';
+$configValues['CONFIG_DB_HOST'] = getenv('RADIUS_DB_HOST');
+$configValues['CONFIG_DB_PORT'] = getenv('RADIUS_DB_PORT');
+$configValues['CONFIG_DB_USER'] = getenv('RADIUS_DB_USER');
+$configValues['CONFIG_DB_PASS'] = getenv('RADIUS_DB_PASS');
+$configValues['CONFIG_DB_NAME'] = getenv('RADIUS_DB_NAME');
 $configValues['CONFIG_DB_TBL_RADCHECK'] = 'radcheck';
 $configValues['CONFIG_DB_TBL_RADREPLY'] = 'radreply';
 $configValues['CONFIG_DB_TBL_RADGROUPREPLY'] = 'radgroupreply';
